scan-triggers/aws-typescript-bucket-full-scan/README.md

```diff
@@ -0,0 +1,51 @@
+# Bucket Full Scan
+
+This is a CDK project to deploy all the required resources to trigger a full scan on a S3 bucket leveraging an existing Trend Micro File Storage Security deployment.
```

Comment:
This is a [CDK](https://docs.aws.amazon.com/cdk/index.html) project to deploy all the required resources to trigger a full scan on a S3 bucket for an existing Trend Micro File Storage Security deployment.